packages/peregrine/lib/talons/CartPage/PriceAdjustments/ShippingMethods/useShippingForm.js

```diff
@@ -32,9 +32,24 @@ export const useShippingForm = props => {
 
     const [
         setShippingAddress,
-        { called: setShippingAddressCalled, loading: isSetShippingLoading }
+        {
+            called: setShippingAddressCalled,
+            error: setShippingAddressError,
```

Comment:
It's a bit counterintuitive to see `setShippingAddressCalled` and `setShippingAddressError` used as object names, since it's our most common format for naming functions.

Consider `calledSetShippingAddress` or `errorSettingShippingAddress`, I suppose?

Reply:
This pattern is used all over the place. If we want to switch it up let's make sure everyone is aware.

Reply:
Renamed in 57ab1a7.